docs/options.md

```diff
@@ -180,14 +180,15 @@ wheels on manylinux2010, and manylinux2014 wheels for any newer Python
 
 > Override the auto-detected target platform
 
-Options: `auto` `linux` `macos` `windows`
+Options: `auto` `linux` `macos` `windows` `pyodide`
 
 Default: `auto`
 
 `auto` will auto-detect platform using environment variables, such as `TRAVIS_OS_NAME`/`APPVEYOR`/`CIRCLECI`.
 
 - For `linux`, you need [Docker or Podman](#container-engine) running, on Linux, macOS, or Windows.
 - For `macos` and `windows`, you need to be running on the respective system, with a working compiler toolchain installed - Xcode Command Line tools for macOS, and MSVC for Windows.
+- For `pyodide` you need to be on an x86 linux and have both Python 3.10 and Python 3.11 installed.
```

Comment:
[x86-64](https://en.wikipedia.org/wiki/X86-64) technically. x86 is 32 bit linux.

Reply:
Updated...